### ISSUE TITLE
[Crash report]  'd3dx9'  

### ISSUE THREAD

**Issue by So2Nik** [bottlesdevs/Bottles#731]
This crash report was generated by Bottles.

**Details**
* **package**: Other
* **version**: 2021.11.14-treviso-3

**Log**
```python3
'd3dx9'  File "/usr/share/bottles/bottles/utils.py", line 317, in __target
    result = self.task_func(*args, **kwargs)

  File "/usr/share/bottles/bottles/backend/manager.py", line 1136, in create_bottle
    _dep = self.supported_dependencies[dep]

```

**Comment by mirkobrombin** [comment]
Should be fixed in 2021.11.14-treviso-4

**Comment by jrDev95** [comment]
Hello, It looks the error persist in treviso-4

```
INFO: Installing dependency [d3dx9] in bottle [Epic].
ERROR: Cannot fetch manifest for d3dx9.
ERROR: Error while running async job: <bound method Manager.create_bottle of <bottles.backend.manager.Manager object at 0x7fc197b9f1f0>>
Exception: 'NoneType' object has no attribute 'set_installed'
  File "/app/share/bottles/bottles/utils.py", line 317, in __target
    result = self.task_func(*args, **kwargs)
  File "/app/share/bottles/bottles/backend/manager.py", line 1143, in create_bottle
    self.dependency_manager.install(config, [dep, _dep])
  File "/app/share/bottles/bottles/backend/dependency.py", line 153, in install
    GLib.idle_add(widget.set_installed, False)
```

The process get freezed or stucked and you need to make a forced quit in the task manager.

I'm using Arch linux and Flatpak 1.12.2

Regards!

**Comment by mirkobrombin** [comment]
Sorry fixed in 2021.11.28
